lib/pages/scout/DesktopScout.dart

```diff
@@ -664,7 +651,7 @@ class _DesktopScoutState extends State<DesktopScout> {
                     // Icon
                     Container(
                         width: 50,
-                        child: Icon(Icons.sports_football,
+                        child: Icon(Icons.sports_baseball,
```

Comment:
this should be determined by a function that will check the Sport property of the AthleteScoutModel and return the proper icon. In fact we can probably just an icon property right to AthleteScoutModel to be set from the Bloc so that all you have to do is call `athlete.sportIcon`